src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+	
+	private final Person person;
+	private final Tag tag;
+	private String operation;
+	
```

Comment:
You might want to use enums for operations. For now even boolean is better. Having it as a String allows for too many arbitrary String values.